open-metadata-implementation/server-chassis/server-chassis-spring/pom.xml

```diff
@@ -172,6 +172,12 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>digital-service-spring</artifactId>
```

Comment:
I would expect you also need to add as an exception in 'ignoredUnusedDeclaredDependency' -- as this is only needed as runtime so our dependency checker will complain you have a dependency not being used without it

Reply:
I'm not actually getting any complaints and it's all running so I'm not really sure how to address this, I'll slack you to clarify.

Reply:
My bad - it is already excluded through the org.odpi.egeria:*spring* pattern match